framework/tests/all-systems/java8/lambda/Lambda.java

```diff
@@ -32,6 +36,7 @@
     Supplier<Integer> f6 = () -> {                 // Complex block body with returns
       if (true) return 12;
       else {
+        new Lambda(System.out::println);           // Use lambda as a constructor argument
```

Comment:
This will actually fail when the I18nChecker runs over all systems because println has a special meaning.  We could define a local println or consume method:
e.g.
static void consume(String s) { }
then instead use Lambda::consume